v1.0/openapi3/parameters/FSPIOP-HTTP-Method.yaml

```diff
@@ -3,4 +3,8 @@ in: header
 schema:
   type: string
 required: false
-description: 'The `FSPIOP-HTTP-Method` header field is a non-HTTP standard field used by the API for signature verification, should contain the service HTTP method. Required if signature verification is used, for more information, see [the API Signature document](https://github.com/mojaloop/docs/tree/master/Specification%20Document%20Set).'
+description: |
+  The `FSPIOP-HTTP-Method` header field is a non-HTTP standard field used by
+  the API for signature verification, should contain the service HTTP method.
+  Required if signature verification is used, for more information, see
+  [the API Signature document](https://github.com/mojaloop/docs/tree/master/Specification%20Document%20Set).'
```

Comment:
```suggestion
  [the API Signature document](https://github.com/mojaloop/docs/tree/master/Specification%20Document%20Set).
```

Reply:
Also, just realized that I actually couldn't open the link... It says 404 not found, is this intended? Or perhaps the document has been moved somewhere else?

Reply:
It has. Lets leave this and the rest of the links for another ticket.

Reply:
Okok 👍

Reply:
Btw, I think the apostrophe could be removed right? Since it was previously used to mark the end of the string, but now we have moved to use multiline description

Reply:
apostrophe removed.